src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2244,6 +2249,15 @@ private function registerNotifierConfiguration(array $config, ContainerBuilder $
             }
         }
 
+        if (class_exists(MercureTransportFactory::class)) {
```

Comment:
Is this check necessary? The transport is always available, isn't it?

Reply:
As far as I know, if the end-user didn't install `symfony/mercure-notifier`, this class won't be available. Therefore, in that case, we don't want to throw any exception for the missing Mercure bundle. Am I wrong?

Reply:
Correct, see similar check above for definition removal